llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -284,6 +286,9 @@ static cl::opt<bool>
                 cat(DwarfDumpCategory));
 static opt<bool> Verify("verify", desc("Verify the DWARF debug info."),
                         cat(DwarfDumpCategory));
+static opt<unsigned> VerifyNumThreads(
+    "verify-num-threads", init(hardware_concurrency().compute_thread_count()),
+    desc("Number of threads to use for --verify."), cat(DwarfDumpCategory));
```

Comment:
Probably best to expand this documentation. 
```
sc("Number of threads to use for --verify. Single threaded verification is the default unless this option is specified. This can cause the output to be non determinisitic, but can speed up verification and is useful when running with the summary only or JSON summary modes.")
```